src/Rings/mpolyquo-localizations.jl

```diff
@@ -1526,6 +1526,25 @@ function intersect(I::MPolyQuoLocalizedIdeal, J::MPolyQuoLocalizedIdeal)
   return L(K)
 end
 
+function intersect(I::MPolyQuoLocalizedIdeal,J::MPolyQuoLocalizedIdeal...)
+  L = base_ring(I)
+  erg = Oscar.pre_image_ideal(I)
+  for K in J
+    base_ring(K) == L || error("base rings must match")
+    erg = intersect(erg,Oscar.pre_image_ideal(K))
+  end
+  return L(erg)
+end
+
+function intersect(VI::Vector{<:MPolyQuoLocalizedIdeal{T}}) where T
+  @assert length(VI) != 0
+  L = base_ring(VI[1])
+  all(J -> base_ring(J) == L,VI) || error("base rings must match")
+  VIpre = [Oscar.pre_image_ideal(J) for J in VI]
```

Comment:
```suggestion
  VIpre = [pre_image_ideal(J) for J in VI]
```

Reply:
And all the other places. `Oscar.` is not necessary. Wondering about why you put them there. 
Do you `julia> use Revise, Oscar` ? for developing Oscar?

Reply:
Yes, I do start it by 'using Revise, Oscar, Test'.
Somehow, non-exported commands erratically show up as not found on my local machine (without changing the julia prompt to yellow or red). Since I could not make sense, I put the Oscar. prefix to be on the safe side. If you say that it is not necessary, I remove it everywhere in this PR.